src/app/github-landing-page/github-landing-page.component.html

```diff
@@ -22,7 +22,7 @@ <h3>Please wait while your GitHub repository installation is being processed. Th
     <hr />
     <h3 class="text-align-center">You can close this tab and return to Dockstore on the original tab.</h3>
 
-    <div class="w-100 cards-container" fxLayout="row" fxLayoutAlign="space-evenly stretch">
```

Comment:
FYI, you could still achieve the resizing by using fxLayout. Something like `fxLayout.lt-sm="column"` which turns the `fxLayout` into a column if the window size is less than "small" (more info here about [sizes](https://github.com/angular/flex-layout/blob/master/docs/documentation/Responsive-API.md))